README.md

```diff
@@ -1,2 +1,11 @@
 # apple_clone
- this is the clone of an apple
+ This is the clone of an old apple website.
```

Comment:
You have made a good effort trying to fix the readme file. It has some important information as required. I believe it could be made better so I will give an example so you could use in other projects

# apple_clone
 This is the clone of an old apple website.
 Specifically this [apple website](http://archive.fo/UW4oR)
 
 This has been made as a part of the [Odin Project.](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients)
 
This is for learning how to use Backgrounds and Gradients.
 
 

 ![alt text](https://i.imgur.com/paAFzXE.png)

## Installation
no installation needed

## Usage
Just pull the repository and view the index.HTML file in a browser


## Contributing
You can put a link to your profile for everyone contributing

## License
Put a license if there is any for this project it could be left out